module/AreaMaster.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file   AreaMaster.cpp
+ * @brief  エリアを攻略するクラス
+ * @author keiya121
+ */
+
+#include "AreaMaster.h"
+
+using namespace std;
+
+AreaMaster::AreaMaster(Area _area, bool _isLeftCourse, bool& _isLeftEdge, int _targetBrightness)
+  : area(_area),
+    isLeftCourse(_isLeftCourse),
+    isLeftEdge(_isLeftEdge),
+    targetBrightness(_targetBrightness)
+{
+}
+
+void AreaMaster::run()
+{
+  const int BUF_SIZE = 128;
+  char buf[BUF_SIZE];  // log用にメッセージを一時保持する領域
+  Logger logger;
+
+  // ファイルから受け取る動作リスト
+  vector<Motion*> motionList;
+
+  // コマンドファイルパスを作成する
+  char commandFilePath[BUF_SIZE];
+  snprintf(commandFilePath, BUF_SIZE, "%s%s%s.csv", basePath,
+           commandFileNames[static_cast<int>(area)], (isLeftCourse ? "Left" : "Right"));
+
+  // 動作インスタンスのリストを生成する
+  motionList = MotionParser::createMotions(commandFilePath, targetBrightness, isLeftEdge);
+
+  // 動作実行のメッセージログを出す
+  snprintf(buf, BUF_SIZE, "\nRun the commands in '%s'\n", commandFilePath);
```

Comment:
```suggestion
  snprintf(buf, SMALL_BUF_SIZE, "\nRun the commands in '%s'\n", commandFilePath);
```
ここらへんも変わるかな。BUF_SIZE = SMALL_BUF_SIZE とかにしても動くと思うけど、他ファイルはSMALL, LARGE のどっちかに合わせてるからそっちでお願い🙏

Reply:
ご確認いただきありがとうございます。確認漏れでした。修正します。